language-agent/JVMMetric.proto

```diff
@@ -85,8 +86,22 @@ enum GCPhrase {
 }
 
 message Thread {
```

Comment:
Let's add a comment here linking to JVM or XBean doc about this.

Reply:
ok.

Reply:
add it.